File: java-main/java-main/Array/Ex1/p5.java
/*Write a program ,take 10 input from the user and print only elements that are divisible by 5.*/
import java.util.*;
class Demo{
        public static void main(String[]args){
                Scanner sc = new Scanner(System.in);
                System.out.println("Enter array size:");
                int size=sc.nextInt();
                int arr[]=new int[size];
                System.out.println("Enter array elements:");
                for(int i=0;i<arr.length;i++){
                        arr[i]=sc.nextInt();
		}
		System.out.println("Number divisible by 5 are: ");
		for(int i=0;i<arr.length;i++){
                        if(arr[i]%5==0){
				System.out.print(arr[i] + " ");
			}
		}
		System.out.println();
        }
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/forloop/p5.c
#include<stdio.h>
void main(){
	int x;


	printf("Enter x\n");
	scanf("%d",&x);

	printf("The factors of %d are:\n",x);

	for(int i=1;i<=x;i++){
		if(x%i==0){
		
			printf("%d\n",i);
		}
	}
}

